src/init.ts

```diff
@@ -2,21 +2,28 @@ import { createRouter } from './core/router';
 import ListPage from './pages/ListPage';
 import ErrorPage from './pages/ErrorPage';
 import { Router } from './core/router/types/router';
+import { render, unmount } from './core/createComponent/index';
+import { VComponent } from './core/createComponent/types/createComponent';
 
 export let router: Router;
 
-export function init(app: HTMLElement): Router {
+export function init(root: HTMLElement): Router {
   // 초기 라우터 설정
-  const routerObj = createRouter({
+  router = createRouter<VComponent>({
     routes: {
-      '/': ListPage({ path: '/' }),
-      '/tech': ListPage({ path: '/tech' }),
-      '/design': ListPage({ path: '/design' }),
+      '/': () => ListPage({ path: '/' }),
+      '/tech': () => ListPage({ path: '/tech' }),
+      '/design': () => ListPage({ path: '/design' }),
       // TODO: '/articles:articleId': detailPage(articleId)}
     },
-    root: app,
-    errorPage: ErrorPage(),
+    root,
+    render: (component: VComponent) => render(component, root),
```

Comment:
```suggestion
    render: (component) => render(component, root),
```

Generic으로 잘 만들어 두셔서 굳이 명시하지 않아도 될 것 같네요!